mobile/lib/components/ra_burger_menu.dart

```diff
@@ -31,10 +31,10 @@ class RaBurgerMenu extends HookWidget {
     (RaRoutes.recordings, 'Nagrania'),
     (RaRoutes.albumOfTheWeek, 'Płyta tygodnia'),
     (RaRoutes.articles, 'Publicystyka'),
-    (RaRoutes.radioPeople, 'Radiowcy'),
+    // (RaRoutes.radioPeople, 'Radiowcy'),
```

Comment:
You're commenting these out, yet in `ra_router_config` you've removed these components' navigation routes entirely - I suppose it'd be better to pick a lane